src/routes/process-user-vocabulary/__tests__/UserVocabularyProcessScreen.spec.tsx

```diff
@@ -31,4 +31,12 @@ describe('UserVocabularyProcessScreen', () => {
     expect(a).toHaveLength(3)
     expect(getByText(getLabels().userVocabulary.creation.addImage)).toBeDisabled()
   })
+  it('should be able to add a recording', async () => {
+    const { findByTestId, getByText } = render(<UserVocabularyProcessScreen navigation={navigation} />)
+    const addAudioButton = getByText(getLabels().userVocabulary.creation.addAudio)
+    expect(addAudioButton).not.toBeNull()
+    fireEvent.press(addAudioButton)
+    const recordButton = await findByTestId('record-audio-button')
+    expect(recordButton).not.toBeNull()
```

Comment:
It does not really make sense to check for notNull if query with "findBy", as the findBy throws an error, if there is no match. You need to use queryBy to get null for no matches.

Reply:
i need the await and findByTestId since its only shown after `setState` adjusted `expect(recordButton).toBeDefined()`